tools/garaga_rs/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
-name = "hades_binding"
+name = "garaga_rs"
 version = "0.1.0"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-pyo3 = { version = "0.15", features = ["extension-module"] }
+pyo3 = { version = "0.15", features = ["extension-module", "num-bigint"] }
 num-bigint = "0.4"
 num-traits = "0.2"
 sha2 = "0.10"
```

Comment:
do you know if the sha2 dependency is really needed ? If we can remove it will be better to have faster compile time and wheel size

Reply:
I will check!